modules/stanford_person_importer/tests/src/Unit/CapTest.php

```diff
@@ -172,7 +172,7 @@ public function testUrls() {
 
     $sunets = implode(',', array_fill(0, 20, 'foo'));
     $url = $this->service->getSunetUrl($sunets);
-    $this->assertEquals("https://cap.stanford.edu/cap-api/api/profiles/v1?uids=$sunets?ps=20", $url);
+    $this->assertEquals("https://cap.stanford.edu/cap-api/api/profiles/v1?uids=$sunets&ps=20", $url);
```

Comment:
We don't want two `?` in the parameter string. I don't even know how this worked at all.

Reply:
Apparently it didn't if the `$count` was greater than 20.  🤷

Reply:
10 is the default from the API, so if there are more than 10 sunets, it would paginate it, so we pass the `ps` parameter to get all sunets on a single page.